SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -1,37 +1,53 @@
-# How to generate a snapshot
+# Snapshots
+A snapshot is a representation of the state of a [SpatialOS world]({{urlRoot}}/content/glossary#spatialos-world) at a given point in time. It stores each [persistent]({{urlRoot}}/content/glossary##persistence) [SpatialOS entity]({{urlRoot}}/content/glossary##spatialos-entity) and the values of their [SpatialOS components]({{urlRoot}}/content/glossary#spatialos-component)' [properties (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#property).
 
-If you're not familiar with snapshots in the context of SpatialOS, please look at the [full snapshot documentation](https://docs.improbable.io/reference/latest/shared/operate/snapshots) (SpatialOS documentation).
+You need a snapshot as the starting point (using an “initial snapshot”) for your [SpatialOS world]({{urlRoot}}/content/glossary##spatialos-world) when you [deploy your game]({{urlRoot}}/content/glossary##deployment) either locally or to the cloud. In the GDK, you need to generate a snapshot when you make certain changes to your project configuration; see [How to generate a snapshot](#how-to-generate-a-snapshot), below.
 
-The GDK snapshots contain three kinds of entities: critical entities, stably named replicated actors and placeholders.
+You can find out more about snapshots in the [SpatialOS snapshot documentation](https://docs.improbable.io/reference/latest/shared/operate/snapshot) but note that this documentation concentrates on working with snapshots using the [SpatialOS SDK]({{urlRoot}}/content/glossary#spatialos-sdk).
 
-### Startup actors
 
-Startup actors are replicated actors that have been placed in a level. We write these startup actors to the snapshot so we can ensure that only one of each entity is spawned when launching multiple server-workers.
+## What’s listed in snapshots
+
+The GDK snapshots contain three kinds of [SpatialOS entities]({{urlRoot}}/content/glossary#spatialos-entities): 
+Startup Actors, critical entities, and placeholder entities.
+
+### Startup Actors
+
+Startup Actors are [stably named replicated Actors (Unreal documentation)](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties/ObjectReferences) that you have placed in a Level. The startup Actors need to be listed in the snapshot so that SpatialOS spawns an entity only once for each Actor. If the startup Actors are not in the snapshot, you may get multiple entities for each Actor when you launch multiple [server-workers]({{urlRoot}}/content/glossary#workers). 
+
+To ensure these Actors are listed in the snapshot, tag them as `[SpatialType]({{urlRoot}}/content/spatial-type)`.
```

Comment:
`[SpatialType]({{urlRoot}}/content/spatial-type)` 
This link is not being displayed correctly because it's wrapped in backticks